examples/advanced/fl_hub/jobs/numpy-cross-val/meta.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "numpy-cross-val",
+  "resource_spec": {},
+  "min_clients" : 2,
+  "deploy_map": {
```

Comment:
You no longer have to define separate apps for the two T2 systems, unless you want them to be different.
You can just define one single app, and put "operators" in the config_fed_server.json.